### DISCUSSION TITLE
Gitignore Update

### DISCUSSION THREAD

**Discussion by Gnayt** [pkgxdev/pkgx#928] [Ideas]
It's about the ossapp, figure it out that the gitignore had some "useless" lines
This one : "electrin/config.json" with an typing error, don't know if it's intented

And we could also add the package-lock, we need to build it anyway.

**Comment by mxcl** [comment]
True. I'll try to remember to add these.

**Reply by Gnayt** [reply]
Created a PR for that https://github.com/pkgxdev/ossapp/pull/850 , not sure if you wanted to still ignore the config.json but added the package-lock
